sdk/python/setup.sh

```diff
@@ -16,8 +16,7 @@ pip install pandas
 
 # <az_ml_sdk_test_install>
 # pip install azure-ai-ml==0.1.0.b8
-pip install azure-ai-ml
 # https://docsupport.blob.core.windows.net/ml-sample-submissions/1905732/azure_ai_ml-1.0.0-py3-none-any.whl
 # </az_ml_sdk_test_install>
-
+pip install https://docsupport.blob.core.windows.net/ml-sample-submissions/3398999/azure_ai_ml-1.13.0-py3-none-any.whl
```

Comment:
revert?

Reply:
This is the candidate which is copied from the pull request shared by Jagadeesh.

Tried to create this pull request to validate if updating the experiment name fixes the issue for import notebooks with this candidate.